### PR TITLE
J=PC-30905 Screenshot/new tab

### PR DESCRIPTION
**Pull request by willespencer**
Add screenshot to created issue and open it in new tab following issue creation

### CODE REVIEW

File: jira.js [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+function jiraSetup(url, image) {
   var xhr = new XMLHttpRequest();
-  createIssue(xhr);
+  createIssue(xhr, url, image);
 }
 
 //Creates an Issue in JIRA
-function createIssue(xhr){
+function createIssue(xhr, url, image){
   var projID = 11300 //CONSULTING ID
```
Comment (sophiakleyman) [inline jira.js:9]: should (some of) these be dynamic? assuming this is just a todo?

File: jira.js [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
-  json = JSON.stringify(createJSON(projID, issueID, accountID));
+  json = JSON.stringify(createJSON(projID, issueID, accountID, url));
 
   xhr.onreadystatechange=function() {
   if (xhr.readyState === 4){   //if complete
-      if(xhr.status === 200){  //check if "OK" (200)
+      if(xhr.status === 201){  //check if "OK" (200)
           console.log("Success", xhr.statusText);
           var result = JSON.parse(xhr.responseText);
           console.log(result);
```
Comment (sophiakleyman) [inline jira.js]: logging

File: jira.js [hunk 3]
```diff
@@ -46,7 +47,7 @@ function createJSON(proj, issue, account){
           // "reporter": {
           //     "name": "tterbush"
           // },
-          "description": "This was a test issue created from a Chrome Extension",
+          "description": "Problem saw at this URL: " + url,
```
Comment (sophiakleyman) [inline jira.js]: nit, but "problem seen"? instead of "saw"

File: jira.js [hunk 4]
```diff
@@ -59,8 +60,8 @@ function printMeta(){
   xhr.open("GET", "https://yexttest.atlassian.net/rest/api/2/issue/createmeta", false);
 
   xhr.onreadystatechange=function() {
-  if (xhr.readyState === 4){   //if complete
-      if(xhr.status === 200){  //check if "OK" (200)
+    if (xhr.readyState === 4){   //if complete
```
Comment (sophiakleyman) [inline jira.js]: this code looks pretty similar to the above, any way we can make it generic?

**Review by sophiakleyman — APPROVED** [review]

File: jira.js (hunk 4 above)
Comment (sophiakleyman) [reply jira.js]: seems like its repeated in a few different places as well

File: jira.js (hunk 1 above)
Comment (willespencer) [reply jira.js:9]: Yeah, I believe projID will always be that value. IssueID changes if it is a bug item (in the following pr) and still need to implement account.

File: jira.js (hunk 4 above)
Comment (willespencer) [reply jira.js]: Simplified slightly in following PR, will work on making it more generic there.

**Pull request merged**